index.js

```diff
@@ -1,93 +1,88 @@
 #!/usr/bin/env node
+
 var chalk = require('chalk');
 var program = require('commander');
-
-var merge = require('merge')
-var exec = require('child_process').exec;
-var jsonfile = require("jsonfile");
-var path = require("path");
+var merge = require('merge');
+var jsonfile = require('jsonfile');
+var path = require('path');
 var tempfile = require('tempfile');
-
 var util = require('util');
+var clim = require('clim');
 
-var clim = require("clim");
-clim.logWrite = function(level, prefixes, msg) {
+clim.logWrite = function (level, prefixes, msg) {
   var pfx = '';
-  if (prefixes.length > 0) pfx = prefixes.join(" ");
+  if (prefixes.length > 0) pfx = prefixes.join(' ');
 
-  level = (f[level.toLowerCase()] || function (p) {return p})(level);
-  line = util.format("%s [%s] %s", pfx, level, msg)
-  process.stderr.write(line + "\n");
+  level = (f[level.toLowerCase()] || function (p) {return p;})(level);
+  var line = util.format('%s [%s] %s', pfx, level, msg);
+  process.stderr.write(line + '\n');
 };
 
-var console = clim(chalk.grey("shield"));
+var console = clim(chalk.grey('shield'));
 
 var f = {
-  error:chalk.red,
+  error: chalk.red,
   link: chalk.underline.blue,
   info: chalk.blue,
   log: chalk.blue,
   warn: chalk.red,
   bold: chalk.bold
-}
+};
 
-var nodeCLI = require("shelljs-nodecli");
+var nodeCLI = require('shelljs-nodecli');
 
-var FirefoxProfile = require("firefox-profile");
-var getPrefs = require("jpm/lib/preferences").getPrefs;
+var FirefoxProfile = require('firefox-profile');
+var getPrefs = require('jpm/lib/preferences').getPrefs;
```

Comment:
Not sure if we want to move these `require()` statements up to the top to keep em grouped. :shrug: